src/modules/flow/iio/iio.json

```diff
@@ -53,7 +53,7 @@
          },
          {
            "data_type": "string",
-           "description": "IIO trigger name. Name of IIO trigger that should be associated to this device for buffered readings. If not set and buffer enabled, will try to use device current trigger, if any. If none, will attempt to create a sysfs trigger",
+           "description": "IIO trigger name. Name of IIO trigger that should be associated to this device for buffered readings. Set to hrtimer:<trigger name> if want to use hrtimer trigger. If not set and buffer enabled, will try to use device current trigger, if any. If none, will attempt to create a sysfs trigger",
```

Comment:
I think support "hrtimer:" will be user-friendly for end users, it will follow the same logic as sysfs trigger: system will create a random trigger name when "NULL or empty" automatically.